release-github/action.yml

```diff
@@ -173,12 +204,54 @@ runs:
       shell: bash
       run: ls -R dist/
 
+    - name: "Install toml"
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip toml==${{ inputs.toml-version }}
+
+    - name: "Get CHANGELOG file location"
+      shell: python
+      run: |
+        import sys
+        sys.path.insert(1, '${{ github.action_path }}/../doc-changelog/')
```

Comment:
This is a very interesting approach. I know that, any file inside a `composite-action/` gets copied at CI/CD runtime. Maybe, this is one step for solving #459. We could have a composite actions that just contain Python files utilities.